pkg/storagev2/storage.go

```diff
@@ -147,3 +148,10 @@ type Store interface {
 	// Delete removes the given Item form the store.
 	Delete(Item) error
 }
+
+// Storage TODO: remove when merged to master.
+type Storage interface {
```

Comment:
So I think it would be best if we define this in the internal packages themselves. This need not be a core interface. This allows better flexibility I think.